src/app/compressed-air-assessment/results-panel/results-panel.component.html

```diff
@@ -49,10 +49,10 @@
                 <app-inventory-table *ngIf="tabSelect == 'current-inventory'"></app-inventory-table>
                 <hr>
```

Comment:
throw an *ngIf="tabSelect == 'current-inventory'" on the hr. don't need the hr above the help text